pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -118,10 +127,14 @@ def add_model_specific_args(parent_parser):
         parser = ArgumentParser(parents=[parent_parser], add_help=False)
         parser.add_argument("--learning_rate", type=float, default=0.0001)
         parser.add_argument("--num_classes", type=int, default=91)
+        parser.add_argument("--backbone", type=str, default=None)
+        parser.add_argument("--fpn", type=bool, default=True)
         parser.add_argument("--pretrained", type=bool, default=False)
         parser.add_argument("--pretrained_backbone", type=bool, default=True)
         parser.add_argument("--trainable_backbone_layers", type=int, default=3)
-        parser.add_argument("--replace_head", type=bool, default=True)
+
+        parser.add_argument("--data_dir", type=str, default=".")
+        parser.add_argument("--batch_size", type=int, default=1)
```

Comment:
Yes, let's discuss #207 as well. I too we should decouple data, transforms, models and keep them independent.

Reply:
@akihironitta where do you suggest we put the data dir and batch size arguments?

Reply:
@ananyahjha93 @oke-aditya In my opinion, `--data_dir` and `--batch_size` args should be added via datamodules, but not via models as previously #206 removed those args from `GAN`.

Reply:
@ananyahjha93 I'd love to know what you think of my suggestion :] (We could probably resolve this later in another PR since the related issue #207 is still an open issue though.)